### ISSUE TITLE
noiseless data

### ISSUE THREAD

**Issue by Stuermer** [megbedell/spectro-pessimism#4]
I assume we don't need noiseless 2d spectra. Is that correct?
Right now, I can generate data of (almost) arbitrary S/N, but since it's still a 'photon-based' generation I can't create 'perfect' data.
Is that OK for our purpose ?

**Comment by davidwhogg** [comment]
Yes, we will never be noiseless. But we do want to get into the effectively Gaussian regime, because that's where SP is proven.
